README.md

```diff
@@ -35,12 +36,26 @@ Nazareth Nuggets is a platform for people to discover new things about Nazareth,
 
 ### [PAS](https://xd.adobe.com/view/6572db59-ff24-4c55-9db7-3142e15141e5/)
 
-Park and Sleep helps tourists navigate unconventional cities like Nazareth. Also this app was built using React library + Redux. [Prototype](https://xd.adobe.com/view/6572db59-ff24-4c55-9db7-3142e15141e5/)
+Park and Sleep, which helps tourists navigate unconventional cities (like Nazareth), was built using React library + Redux. 
 
+I used the following technologies:
 - React
 - Express
 - MongoDB
 - Webpack + Babel + Axios
 
-# Contact
+## EXPERIENCE
```

Comment:
@rebeccaradding just changed can this be merged or is there other stuff to add ?